### ISSUE TITLE
dGPU driver installation failed

### ISSUE THREAD

**Issue by darshanhuang1** [intel-analytics/ipex-llm#11568]
OS: 22.04.3 LTS, kernel default 6.5.0-41-general x86_64
When install dGPU driver following this guide (https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_linux_gpu.html), always counter fail message as diagram showed. Even upgrade to 6.5.11, also failed. 

![MicrosoftTeams-image](https://github.com/user-attachments/assets/c915eb0b-90af-4c31-a4fc-69f2e393f433)

**Comment by liu-shaojun** [comment]
Hi @darshanhuang1 
Could you please provide the output of `sudo apt list --installed | grep linux-headers`?

**Comment by liu-shaojun** [comment]
Hi @darshanhuang1 

We have verified this guide on kernel 6.5.0-35, could you try to use kernel 6.5.0-35?

**Comment by darshanhuang1** [comment]
Ok. And how about 6.2 kernel? What versions have you verified?

**Comment by liu-shaojun** [comment]
We have verified 6.2.0-39-generic.

**Comment by liu-shaojun** [comment]
Hi @darshanhuang1,

We followed the instructions from [this guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_linux_gpu.html) to install the GPU driver on kernel 6.5.0-41-generic. We found that `intel-i915-dkms=1.24.2.17.240301.20+i29-1` encounters the same error, but upgrading to the latest version `1.24.3.23.240419.26+i30-1` resolves the issue. 

If you are still using kernel 6.5.0-41-generic, you can execute the following commands to check if you can install the latest version:

```
sudo apt update
sudo apt-cache policy intel-i915-dkms
```

If you have the older version installed, remove it first and then install the latest one. If you cannot find the latest version in the candidate list, there might be an issue with the apt source.

```
sudo apt purge -y intel-i915-dkms
sudo apt install -y intel-i915-dkms 
```

Let me know if you need any further adjustments!

**Comment by darshanhuang1** [comment]
yes, work on 6.5.0-41-generic, will try anoter version soon. Thanks!

**Comment by darshanhuang1** [comment]
Tried 6.6 kernel but failed. Could the driver be supported on kernel 6.6?

**Comment by liu-shaojun** [comment]
We have currently only validated the driver on kernel versions 6.2 and 6.5. We have not yet tested it on kernel 6.6, so we are unable to confirm its compatibility with that version at this time.
